server/jwt/jwt.go

```diff
@@ -0,0 +1,162 @@
+package jwt
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/fabiocicerchia/go-proxy-cache/config"
+	"github.com/lestrrat-go/backoff/v2"
+	"github.com/lestrrat-go/jwx/jwk"
+	"github.com/lestrrat-go/jwx/jwt"
+	log "github.com/sirupsen/logrus"
+)
+
+var co *config.Jwt
+var jwtKeyFetcher *jwk.AutoRefresh
+
+func InitJWT(conifg *config.Jwt) {
```

Comment:
fixing a typo here, although need to change all the occurrences of it.

```suggestion
func InitJWT(config *config.Jwt) {
```